examples/quick-start-opensearch-nodejs/tracetest/opensearch/opensearch-analytics.yaml

```diff
@@ -0,0 +1,22 @@
+entry-pipeline:
+  delay: "100"
+  source:
+    otel_trace_source:
+      ssl: false
+  sink:
+    - pipeline:
+        name: "raw-pipeline"
+
+raw-pipeline:
+  source:
+    pipeline:
+      name: "entry-pipeline"
+  prepper:
+    - otel_trace_raw_prepper:
+  sink:
+    - opensearch:
+        hosts:
+          - "http://opensearch:9200"
+        insecure: true
+        trace-analytics-raw: true
+        index: traces
```

Comment:
Maybe worth saying that this is the value you have to use in your [OpenSearch data store configuration](https://github.com/kubeshop/tracetest/pull/1593/files#diff-51976d6e7a4b39e829880ea12c03ee5c494713492f830e0e830735139bab9816R22).